src/Simulation/GroundStation/GroundStation.cpp

```diff
@@ -63,7 +63,7 @@ bool GroundStation::CalcIsVisible(const Vector<3> sc_pos_ecef_m) {
   trans_mat_ecef_to_ltc[2][2] = sin(lat);
 
   Vector<3> sc_pos_ltc = trans_mat_ecef_to_ltc * (sc_pos_ecef_m - gs_position_ecef_);  // Satellite position in LTC frame [m]
-  Vector<3> dir_sc_pos_ltc = normalize(sc_pos_ltc);
+  sc_pos_ltc = normalize(sc_pos_ltc);
```

Comment:
これ，
- `normalize()`の戻り値をとりあえず変数に入れていたが，別に使っていなかった
- `normalize()`はconstでない参照を取り，引数と同じものを返す

ってことですかね？

Reply:
そういうことっぽいな
https://github.com/ut-issl/s2e-core/blob/62af3e29927cb5db8a564438246be2687dd74a59/src/Library/math/Vector_tfs.hpp#L125-L139

Reply:
そもそも`normalize()`が`v`を返すのがおかしいかんじがしますね(const参照を受け取って新しく構築したVectorを返すならまだしも)

Reply:
そのとおりで、本当はnormalize修正したいんですよね。。。ただ、色んな場所に影響あるので手が出しづらいという感じです。数学ライブラリ全部見直してきれいにしたいなーと思っているのですが。。。

Reply:
ですねー > 影響範囲がデカい
まあそもそもEigenにしたいみたいな話もありますし，(v6.0.0とかで)breaking change的にやることリスト作っておいてもよさそうです．

Reply:
というのはさておき，これは戻り値が新しく構築されているのではなく引数が破壊的に変更されている，というのを分かりやすくするためにもこんなかんじでいいのでは，という気がします．
```suggestion
  normalize(sc_pos_ltc);
```

Reply:
Thank you for your suggestion. I fixed there.